python/sglang/srt/managers/scheduler.py

```diff
@@ -513,7 +530,9 @@ def print_decode_stats(self):
         )
         throughput = self.num_generated_tokens / (time.time() - self.last_stats_tic)
         self.num_generated_tokens = 0
-        self.last_stats_tic = time.time()
+        # self.last_stats_tic = time.time()
```

Comment:
Because `last_stats_time` will be used in get_stats for `TPOT` [code](https://github.com/Lzhang-hub/sglang/blob/3672b1a13ac8170e01cbb7115338821829cc80bf/python/sglang/srt/managers/scheduler.py#L865), If update when print decode log, it will affect the calculation of `TPOT`.

I think last_stats_time should be updated after each round of iteration, Is there any bug in updating it at [line 360](https://github.com/Lzhang-hub/sglang/blob/3672b1a13ac8170e01cbb7115338821829cc80bf/python/sglang/srt/managers/scheduler.py#L360)?

Reply:
Oh, comment seems to affect the decode `gen throughput (token/s)` log, I will check it.

Reply:
@ByronHsu  Because the original decode log is not printed every iter, it is on condition `self.forward_ct_decode % self.server_args.decode_log_interval == 0`. 
There are two solutions to this problem:
1. Print throughput for every iter. 
2. Add a new variable to record the time of printing the throughput  log. 

Which do you think is better?

Reply:
let's take the 2nd approach. the first one will introduce much overhead